src/annotator/frame-observer.ts

```diff
@@ -47,20 +57,16 @@ export class FrameObserver {
     this._mutationObserver.disconnect();
   }
 
-  /**
-   * @param {HTMLIFrameElement} frame
-   */
-  async _addFrame(frame) {
+  private async _addFrame(frame: HTMLIFrameElement) {
     this._annotatableFrames.add(frame);
     try {
       await onNextDocumentReady(frame);
       if (this._isDisconnected) {
         return;
       }
       const frameWindow = frame.contentWindow;
-      // @ts-expect-error
```

Comment:
Huh, I'm curious what the error was...was it because `frameWindow` might be undefined?

Reply:
Yes, exactly.

![image](https://user-images.githubusercontent.com/2719332/228765903-8303c1fd-5f12-47fa-9cd5-a32421789b6f.png)

I have addressed it by adding `!`, which is what we have been doing in other places.